source/includes/fundamentals/code-snippets/Geo.java

```diff
@@ -63,13 +70,20 @@ private void rangeExample() {
         MongoCollection<Document> collection = database.getCollection("theaters");
         // begin rangeExample
 
-        // code to set up your mongo collection ...
+        // Add your MongoCollection setup code here
+        // Creates a Polygon instance for a geolocation query
```

Comment:
Suggestion:
I think it would be better to omit the part of this comment that just describes the code ("Polygon instance").

E.g.
"Creates a set of points that defines the bounds of a geospatial shape"